src/test/java/org/mockitousage/matchers/CapturingArgumentsTest.java

```diff
@@ -298,7 +313,7 @@ public void captures_correctly_when_captor_used_multiple_times() throws Exceptio
 
         // then
         // this is only for backwards compatibility. It does not make sense in real to do so.
-        verify(mock).mixedVarargs(any(), argumentCaptor.capture(), argumentCaptor.capture(), argumentCaptor.capture());
+        verify(mock).mixedVarargs(any(), argumentCaptor.capture());
```

Comment:
I think the commits need to be merged by you once we are done with the review/improvement cycle. The mockito guys are short in time.
About the new changes: Maybe we can simplify the new code by using getAruments instead of getRawArguments.  We need some more tests for: more ArgumentMatcher than Arguments and less ArgumentMatcher that Arguments. I cant provide Code snippets now, writing on a Smartphone which is a pain.